src/behaviors/TestableResponseBehavior.php

```diff
@@ -12,8 +12,20 @@
  */
 class TestableResponseBehavior extends Behavior {
 
+    public TestableResponse $response;
```

Comment:
Instead of the new property, why not just a class docblock to type `$owner` as a `TestableResponse`? I'm not against this approach, just curious if we need the extra property.

Reply:
For clearity only